benchmark/scenarios/test/test_read_request_load.py

```diff
@@ -216,6 +218,37 @@ def test_scenario_throws_exception_when_rate_drops(self, _logger):
         self.assertIsInstance(failure.value, RequestRateTooLow)
 
     @capture_logging(None)
+    def test_scenario_succeeds_when_rate_has_tolerated_drop(self, _logger):
+        """
+        ``read_request_load_scenario`` succeeds even if the rate drops,
+        if it is within the tolerance percentage.
+
+        Establish the requested rate by having the ``FakeFlockerClient``
+        respond to all requests, then lower the rate by dropping
+        alternate requests.
+        """
+        c = Clock()
+
+        cluster = self.make_cluster(RequestDroppingFakeFlockerClient)
+        sample_size = 5
+        s = read_request_load_scenario(c, cluster, sample_size=sample_size,
+                                       tolerance_percentage=0.6)
+        cluster.get_control_service(c).drop_requests = True
+        d = s.start()
+
+        c.pump(repeat(1, sample_size))
+
+        cluster.get_control_service(c).drop_requests = True
```

Comment:
I don't see the purpose of lines 241-243. They look like they might be a copy/paste mistake? Or maybe they are needed, but certainly line 241 would merit a further explanation coment. Looking at RequestDroppingFakeFlockerClient, I don't see how drop_requests would become unset, but I'm pretty new to this code.